index.bs

```diff
@@ -2421,6 +2415,106 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorMakeCredential=] operation.
 
 
+## Authenticator taxonomy ## {#sctn-authenticator-taxonomy}
+
+Different kinds of [=authenticators=] enable different use cases along two independent dimensions: the [=authentication ceremony
+structure=] and the [=authenticator=]'s [=attachment modality=]. The [figure below](#fig-authData) illustrates how the use cases
+relate to different kinds of [=authenticators=].
+
+<figure id="fig-authenticator-taxonomy">
+    <img src="images/authenticator-taxonomy.svg"/>
+    <figcaption>Relationships between [=authenticator=] properties and use cases.</figcaption>
+</figure>
+
+The terms [=second-factor use case|second-factor=], [=first-factor use case|first-factor=], [=single-step use case|single-step=]
+and [=username-less use case|username-less=] referenced in the figure refer to the use cases defined in
+[[#sctn-authentication-ceremony-structures]]. The terms [=platform authenticator|platform=] and [=roaming authenticator|roaming=]
+are defined in [[#sctn-authenticator-attachment-modality]]. The terms [=user verification=] and [=client-side-resident credential
+private key=] are defined in [[#terminology]].
+
+
+### <dfn>Authentication ceremony structures</dfn> ### {#sctn-authentication-ceremony-structures}
+
+The [=authentication ceremony structure=] is defined by the [=authentication factors=] used in the [=authentication
+ceremony|ceremony=]. Different [=authentication ceremony structures=] provide different benefits for security and usability.
+
+The following discussion assumes that the [=[RP]=] can verify the security guarantees of the [=authenticator=] by inspecting
+[=attestation statements=] generated by the [=authenticator=]. In particular, the [=[RP]=] cannot trust that the [=authenticator=]
+can verify [=something you know=] or [=something you are=] unless these properties can be derived from the [=attestation
+statement=]. Thus, an [=authenticator=] that cannot provide such an [=attestation statement=] can only enable the [=second-factor
+use case=].
```

Comment:
The last two sentences here hark back to our recent discussion wrt "trust" and attestation or the lack of it: https://github.com/w3c/webauthn/pull/899#discussion_r195171633. 

The first sentence also given our default of `AttestationConveyancePreference <- none`

I'm hesitant to get into this "trust" stuff here.  maybe we do need an explainer doc, or appendicies.

Reply:
Yeah, agreed. See https://github.com/w3c/webauthn/pull/960

Reply:
Fixed in #960.